src/calendar/Calendar.ts

```diff
@@ -16,7 +16,6 @@ import * as iconCss from '../common/styles/icons.m.css';
  *
  * Properties that can be set on a Calendar component
  *
- * @property CustomDateCell    Custom widget constructor for the date cell. Should use CalendarCell as a base.
```

Comment:
Did you remove sub-widgets as part of this PR as well?

Reply:
The problem is sub-widgets make it much easier to handle data binding on event handlers (e.g. `onClick` on a specific date cell returns the correct date and disabled state). Based on talking with Ant, I think the compromise reached was to leave in sub-widgets, but try not to expose them to the user

Reply:
I've been wondering if it makes sense to remove the typedoc for CalendarCell. It's marginally useful to developers wanting to extend CalendarCell, but I definitely don't want it showing up in Dojo documentation